### PR TITLE
Add export tickets to fabric ticketing

### PR DESCRIPTION
**Pull request by ssalinas24**
Jira: https://rackspace.atlassian.net/browse/PLATEXP-140

This PR adds Fabric Ticketing documentation for the  Export Tickets feature.

### CODE REVIEW

**Comment by ssalinas24** [comment]
@asmita32675 Please review this PR when you have a chance. Leave comments for any lines that I should fix or update. Thank You!

File: content/how-to/rackspace-fabric/fabric-ticketing/index.md
Comment (asmita32675) [inline content/how-to/rackspace-fabric/fabric-ticketing/index.md:67]: Make Refresh List as bold, since this is a button on the UI.

File: content/how-to/rackspace-fabric/fabric-ticketing/index.md
Comment (asmita32675) [inline content/how-to/rackspace-fabric/fabric-ticketing/index.md:70]: Need to change "Close Tickets" to "Close Selected". As we click on Actions button, we could see in the drop down list Export Tickets and Close Selected. So our nomenclature in document should match with UI.

File: content/how-to/rackspace-fabric/fabric-ticketing/index.md
Comment (asmita32675) [inline content/how-to/rackspace-fabric/fabric-ticketing/index.md:81]: Need to delete period and csv should be CSV.

File: content/how-to/rackspace-fabric/fabric-ticketing/index.md
Comment (asmita32675) [inline content/how-to/rackspace-fabric/fabric-ticketing/index.md:85]: Need to delete colon from the heading. Also same change for Left Side Ticket Panel.

File: content/how-to/rackspace-fabric/fabric-ticketing/index.md
Comment (asmita32675) [inline content/how-to/rackspace-fabric/fabric-ticketing/index.md:77]: Change tickets to Tickets

File: content/how-to/rackspace-fabric/fabric-ticketing/index.md
Comment (asmita32675) [inline content/how-to/rackspace-fabric/fabric-ticketing/index.md:79]: **Pending Customer** instead  **Pending Tickets** as per UI

File: content/how-to/rackspace-fabric/fabric-ticketing/index.md
Comment (ssalinas24) [reply content/how-to/rackspace-fabric/fabric-ticketing/index.md:85]: @asmita32675  If you search the code base for `Warning`, you will notice that tons of files have the colon following it. However, it is sometimes within the bold like this `**Warning:**` and some times it is outside the bold like this `**Warning**:`. So I figured we needed it.

File: content/how-to/rackspace-fabric/fabric-ticketing/index.md
Comment (ssalinas24) [reply content/how-to/rackspace-fabric/fabric-ticketing/index.md:77]: @asmita32675 `Open Tickets` and `Closed Tickets` are the actual titles of the options. The third option is titled `Pending Customer`, so I left the following `tickets` word lowercase. Do you still feel I need to capitalize `tickets`?

File: content/how-to/rackspace-fabric/fabric-ticketing/index.md
Comment (asmita32675) [reply content/how-to/rackspace-fabric/fabric-ticketing/index.md:85]: Agree.

**Comment by ssalinas24** [comment]
@asmita32675 I have addressed the feedback. Let me know if I need other changes. If everything looks good, then I just need your approval. You would need to click the green Review button on the top right of page, and select the approve radio option when submitting the review.

**Review by asmita32675 — APPROVED** [review]

**Review by lfielder — APPROVED** [review]

**Pull request merged**
